packages/gestalt/src/IconCompact.tsx

```diff
@@ -0,0 +1,101 @@
+import InternalIcon from './Icon/InternalIcon';
+import compactIconsVR from './icons-vr-theme/compact/index';
+
+export type IconColor =
+  | 'default'
+  | 'disabled'
+  | 'subtle'
+  | 'success'
+  | 'error'
+  | 'warning'
+  | 'info'
+  | 'recommendation'
+  | 'inverse'
+  | 'shopping'
+  | 'brandPrimary'
+  | 'light'
+  | 'dark';
+
+type Props = {
+  /**
+   * Label for screen readers to announce Icon. This populates the `aria-label` attribute. If the label is hidden, use an empty string for the label (`accessibilityLabel=""`) to set `aria-hidden`.
+   *
+   * See the [Accessibility guidelines](https://gestalt.pinterest.systems/web/icon#Accessibility) for details on proper usage.
+   */
+  accessibilityLabel: string;
+  /**
+   * The colors available to apply to Icon.
+   *
+   * See the [color variant](https://gestalt.pinterest.systems/web/icon#Colors) to learn more.
+   */
+  color?: IconColor;
+  /**
+   * Available for testing purposes, if needed. Consider [better queries](https://testing-library.com/docs/queries/about/#priority) before using this prop.
+   */
+  dataTestId?: string;
+  /**
+   * SVG icon from the Gestalt icon library to use within Icon.
+   *
+   * See the [icon library](https://gestalt.pinterest.systems/foundations/iconography/library) to explore available options.
+   */
+  icon?: keyof typeof compactIconsVR;
+  /**
+   * Defines a new icon different from the built-in Gestalt icons.
+   *
+   * See the [custom icon](https://gestalt.pinterest.systems/web/icon#Custom-icon) variant to learn more.
+   */
+  dangerouslySetSvgPath?: {
+    __path: string;
+  };
+  /**
+   * Properly positions Icon relative to an inline element, such as Text using the inline property.
+   */
+  inline?: boolean;
+  /**
+   * Use a number for pixel sizes or a string for percentage based sizes.
+   *
+   * See the [size](https://gestalt.pinterest.systems/web/icon#Size) variant to learn more.
+   */
+  size?: number | string;
```

Comment:
should we limit size here?

Reply:
they should be '16' the only accepted value for now

Reply:
No I think this size is different afaik. This is the SVG height/width. But what we mean by compact Icons is that they are thinner (they can in theory be scaled up) to larger viewboxes.

![image](https://github.com/user-attachments/assets/31463222-f640-4a9e-ac2a-f3cd8c1d9a28)

Reply:
But maybe you are right in that they should be fixed

Reply:
Removed